app/assets/stylesheets/cm_admin/base/table.scss

```
@@ -77,20 +77,23 @@
       overflow: auto;
       margin-top: 40px;
       margin-bottom: 50px;
-      height: 280px;
+      height: calc(100vh - 365px);
+      // height: 280px;
       .cm-table {
         table-layout: fixed;
         position: relative;
         min-width: fit-content;
         border-collapse: collapse;
         border: 1px solid $grey-lighter-clr;
+        border-top-width: 0;
         border-radius: $radius-8;
         &__header {
           .header-row {
             display: block;
             width: 100%;
             position: relative;
             box-shadow: inset 0px -1px 0px rgba(148, 151, 155, 0.15);
+            border-top: 1px solid #F3F4F6;
```

@barath46 - Can we use the SCSS variable instead?
```suggestion
            border-top: 1px solid $grey-lighter-clr;
```